node/nodejs_intro/README.md

```diff
@@ -1,65 +1,98 @@
 # Node.js Intro + The Event Loop
 
-# Topics
-- What is Node.js?
-- Diving deeper into Programming Languages
-- Google's Javascript Engine: V8
-- Know and understand the event loop.
-- Know the different phases of the event loop.
-
-
-# 1. What is Node.js?
-
-### Why not just call it Javascript for Server?
+# Objectives
+- `Understand the need for higher-level programming languages`
+- `Articulate how Javascript and Node.js are related, but different`
+- `Demonstrate the different phases of the event loop`
+
+## Key Vocabulary
+- Higher-level programming
```

Comment:
I would add:
- Coding environment
 
And take a moment to define it. Something like: Firefox vs Chrome vs Node vs your computer vs my computer - are all different environments that run JS.

Reply:
Good call! Will add :)